Makefile.config.in

```diff
@@ -221,6 +221,13 @@ NAKED_POINTERS=false
 
 # Deprecated variables
 
+## Variables deprecated since OCaml 5.3
+
+OCAMLC_CFLAGS=$(BYTECODE_CFLAGS)
+OCAMLOPT_CFLAGS=$(OCAMLOPT_CFLAGS)
+OCAMLC_CPPFLAGS=$(BYTECODE_CPPFLAGS)
+OCAMLOPT_CPPFLAGS=@native_cppflags@
```

Comment:
It seems OK to have `OCAMLC_CFLAGS` and `OCAMLC_CPPFLAGS` being renamed, and `OCAMLOPT_CFLAGS` and `OCAMLOPT_CPPFLAGS` removed. There's a small typo with it:

```suggestion
OCAMLC_CFLAGS=$(BYTECODE_CFLAGS)
OCAMLOPT_CFLAGS=@native_cflags@
OCAMLC_CPPFLAGS=$(BYTECODE_CPPFLAGS)
OCAMLOPT_CPPFLAGS=@native_cppflags@
```

(I equally don't mind if you just go with the first commit)